content/fr/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -0,0 +1,355 @@
+ses pods pour correspondre à ce nombre.
+
+Si vous ne spécifiez pas `.spec.replicas`, la valeur par défaut est 1.
+
+## Travailler avec des ReplicaSets
+
+### Suppression d'un ReplicaSet et de ses pods
+
+Pour supprimer un ReplicaSet et tous ses pods, utilisez [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete). The [Garbage collector](/docs/concepts/workloads/controllers/garbage-collection/) supprime automatiquement tous les pods associés par défaut.
+
+Lors de l’utilisation de l’API REST ou de la bibliothèque `client-go`, vous devez définir` propagationPolicy` sur `Background` ou` Foreground` dans
+l'option -d.
+Par exemple :
+```shell
+kubectl proxy --port=8080
+curl -X DELETE  'localhost:8080/apis/extensions/v1beta1/namespaces/default/replicasets/frontend' \
+> -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Foreground"}' \
+> -H "Content-Type: application/json"
+```
+
+### Supprimer juste un ReplicaSet
+
+Vous pouvez supprimer un ReplicaSet sans affecter ses pods à l’aide de [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete) avec l'option `--cascade=false`.
+Lorsque vous utilisez l'API REST ou la bibliothèque `client-go`, vous devez définir `propagationPolicy` sur `Orphan`.
+Par exemple :
+```shell
+kubectl proxy --port=8080
+curl -X DELETE  'localhost:8080/apis/extensions/v1beta1/namespaces/default/replicasets/frontend' \
+> -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Orphan"}' \
+> -H "Content-Type: application/json"
+```
+
+Une fois l’original supprimé, vous pouvez créer un nouveau ReplicaSet pour le remplacer. Tant que l'ancien et le nouveau `.spec.selector` sont identiques, le nouveau adoptera les anciens Pods.
+Cependant, le ReplicaSet ne fera aucun effort pour que les pods existants correspondent à un nouveau Pod template.
+Pour mettre à jour les Pods à une nouvelle spec de manière contrôlée, utilisez un
+[Deployment](/docs/concepts/workloads/controllers/deployment/#creating-a-deployment), car les ReplicaSets ne supportent pas de rolling update directement.
+
+### Isoler les pods d'un ReplicaSet
+
+Vous pouvez supprimer les pods d'un ReplicaSet en modifiant leurs labels. Cette technique peut être utilisée pour enlever les pods
+pour le débogage, récupération de données, etc. Les pods ainsi supprimés seront automatiquement remplacés 
+(en supposant que le nombre de réplicas n’est pas également modifié).
+
+### Scaling d'un ReplicaSet
+
+Un ReplicaSet peut facilement être scalé en mettant simplement à jour le champ `.spec.replicas`. Le contrôleur ReplicaSet
+garantit que le nombre souhaité de pods avec un sélecteur de label correspondant soient disponibles et opérationnels.
+
+### ReplicaSet en tant que Horizontal Pod Autoscaler Target
+
+Un ReplicaSet peut également être une cible pour
+[Horizontal Pod Autoscalers (HPA)](/docs/tasks/run-application/horizontal-pod-autoscale/). 
```

Comment:
Si on traduit Horizontal Pod Autoscaler?

Reply:
As it's a Kubernetes concept I'm not sure if it's worth translating
Furthermore, Horizontal => Horizontal, Pod => Pod and we usually directly use the word `scale`   even in French.

@remyleone if you have any inputs?

Reply:
Je pense qu'il faut laisser HPA car on a pas de mot direct en français aussi bien pour Pod et pour autoscaler. Donc ça n'a pas beaucoup d'intérêt de le traduire